packages/itwin/grouping-mapping-widget/src/widget/components/CustomCalculationAction.tsx

```diff
@@ -227,9 +231,25 @@ export const CustomCalculationAction = ({
           />
           <LabeledTextarea
             value={formula}
-            required
             name='formula'
-            label='Formula'
+            label={
+              <div className='gmw-formula-information-icon'>
+                <Label required>
+                  Formula
+                </Label>
+                <Tooltip
+                  content=' Formulas can be used to create calculated properties based on already
```

Comment:
Is it possible to include a link to the formula documentation? 
https://developer.bentley.com/apis/insights/operations/create-customcalculation/

Maybe we could even shorten the text of the tooltip to something that simply directs users to somehow navigate to the documentation page where everything is explained much more elaborately.

Reply:
Maybe clicking on the information icon could take you to the docs? Not sure if that is a UX antipattern for tooltips _shrugs_. The icon already has the mouse hover click cursor... soooo not too much of a stretch to make a click action.
@Guillar1

Reply:
I don't see anything wrong with making it open a new tab to the docs.

Reply:
done!

Reply:
https://github.com/iTwin/viewer-components-react/assets/22688247/17c5eab3-1560-4600-8cc8-561ccd3880c6